### DISCUSSION TITLE
Change the map filter "By difficulties" to the same than osu stable

### DISCUSSION THREAD

**Discussion by MidKnightXI** [ppy/osu#17469] [Ideas] [answered]
The map filtering "By difficulties" UI/UX on Lazer has always been terrible compared to stable.

It's painful to search for a map with the difficulty that you want so would it be possible to bring back the "By difficulties" from stable?

![image](https://user-images.githubusercontent.com/35759490/160258762-4512ec7c-99cf-413e-a541-6674b0855a48.png)

**Comment by bdach** [comment, marked as answer]
planned: https://github.com/ppy/osu/issues/4919 https://github.com/ppy/osu/issues/2547
